jwst/saturation/saturation_step.py

```diff
@@ -10,43 +9,52 @@
 
 
 class SaturationStep(Step):
-    """
-    This Step sets saturation flags.
-    """
+    """Sets saturation flags."""
 
     class_alias = "saturation"
 
     spec = """
         n_pix_grow_sat = integer(default=1) # number of layers adjacent pixels to flag
         use_readpatt = boolean(default=True) # Use grouped read pattern information to assist with flagging
-    """ # noqa: E501
+    """  # noqa: E501
 
-    reference_file_types = ['saturation', 'superbias']
+    reference_file_types = ["saturation", "superbias"]
 
     def process(self, step_input):
-
+        """
+        Set the saturation flags.
+
+        Parameters
+        ----------
+        step_input : DataModel or str
+            Input datamodel or string name of the fits file.
+
+        Returns
+        -------
+        result : DataModel
+            Result of setting the saturation flags to saturated pixels to the data model.
```

Comment:
This phrasing is a little awkward - maybe this instead?
```suggestion
            Output datamodel with saturation flags set for saturated pixels.
```